src/cascadia/TerminalApp/AppLogic.h

```diff
@@ -57,7 +57,7 @@ namespace winrt::TerminalApp::implementation
         // the ctor, you're going to have a bad time. It'll mysteriously fail to
         // activate the AppLogic.
         // ALSO: If you add any UIElements as roots here, make sure they're
-        // updated in _AppLogiclyTheme. The root currently is _root.
+        // updated in _ApplyTheme. The root currently is _root.

```

Comment:
I'm fairly certain this is a correct fix... There's no sign of the string I'm changing.

Reply:
Ugh. This was my fault. It was App --> AppLogic in an overzealous find/replace. I thought I fixed them all. So yeah. `Apply` is the correct revert.